### PR TITLE
#623 Methods that throw exceptions are valid non-static methods

### PR DESCRIPTION
**Pull request by gumbelmj**
#623 Methods that throw exceptions are valid non-static methods. Updated the check and verified it with a change to the test.

### CODE REVIEW

**Comment by davvd** [comment]
@gumbelmj Let me find a reviewer for this pull request, thanks for submitting it

**Comment by davvd** [comment]
@krzyk the pull request is rather small, pls review it

**Comment by krzyk** [comment]
@gumbelmj Please add reference to the issue in the PR description (the #623)

File: qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/NonStaticMethodCheck/Valid.java [hunk 1]
```diff
@@ -34,4 +34,10 @@ public void someOverrideMethod() {
 
     // this method is not static, but it is native
     public native void someNativeMethod();
+
```
Comment (krzyk) [inline qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/NonStaticMethodCheck/Valid.java]: @gumbelmj remove this empty line

**Comment by krzyk** [comment]
@gumbelmj Two comments above

**Comment by gumbelmj** [comment]
@krzyk I removed the extra line and added the ticket number to the description

**Comment by krzyk** [comment]
@gumbelmj thanks

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by krzyk** [comment]
@gumbelmj now you have to ask issue creator to close the issue

**Comment by krzyk** [comment]
@rultor deploy pls

**Comment by davvd** [comment]
@rultor deploy now pls

**Comment by davvd** [comment]
@krzyk thanks, paid, **22 mins** to your account, payment ID is `76502130`, task took 5 hours and 6 mins to complete. extra minutes for review comments (c=7). +22 added to your rating, at the moment it is: [+5067](http://www.netbout.com/b/35035?open=rating)

**Pull request merged**